tick/simulation/features.py

```diff
@@ -25,17 +26,20 @@ def features_normal_cov_uniform(n_samples: int = 200, n_features: int = 30):
         n_samples realization of a Gaussian vector with the described
         covariance
     """
-    C = np.random.uniform(size=(n_features, n_features))
+    C = np.random.uniform(size=(n_features, n_features), dtype=dtype)
     np.fill_diagonal(C, 1.0)
     cov = 0.5 * (C + C.T)
-    return np.random.multivariate_normal(
+    ndarray = np.random.multivariate_normal(
         np.zeros(n_features), cov, size=n_samples)
+    if dtype != np.float64:
+        return ndarray.astype(dtype)
+    return ndarray
 
 
 def features_normal_cov_toeplitz(n_samples: int = 200, n_features: int = 30,
-                                 cov_corr: float = 0.5):
+                                 cov_corr: float = 0.5, dtype=np.float64):
```

Comment:
We should normalize the default parameters : let's use strings everywhere or np.dtypes everywhere. Above was strings, so I suggest dtype='float64'